tensorboard/plugins/debugger_v2/tf_debugger_v2_plugin/views/alerts/alerts_component.css

```diff
@@ -15,13 +15,24 @@ limitations under the License.
 
 .alerts-breakdown-container {
   font-size: 13px;
-  padding: 10px 10px 10px 50px;
+  padding: 10px 10px 10px;
   position: relative;
 }
 
+.alerts-breakdown-type {
+  border-radius: 0px 10px 10px 0px;
```

Comment:
0px -> 0. https://google.github.io/styleguide/htmlcssguide.html#0_and_Units

Reply:
Done here and at another place in the same file. Thanks.